src/builtins/builtin_export.c

```diff
@@ -21,15 +21,16 @@ static void	invalid_identifier_message(char *name)
 	ft_putstr_fd("': not a valid identifier\n", STDERR_FILENO);
 }
 
-int	builtin_export(char **argv)
+int	builtin_export(int argc, char **argv)
 {
 	int		res;
 	char	**split_res;
 
-	res = 0;
-	if (argv == NULL)
+	if (argv == NULL || argv[0] == NULL)
 		return (FT_ERROR);
-	if (argv[0] == NULL)
+	(void)argc;
+	res = 0;
+	if (argv[1] == NULL)
```

Comment:
argv is used after so should be incremented it by 1

Reply:
Fixed.